scripts/lib/mkcloud-onhost.sh

```diff
@@ -121,7 +121,7 @@ function onhost_cacheclouddata
         for a in $architectures; do
             local suffix
             # Mirror SLES/HA/SOC update + pool repos
-            for suffix in Pool Updates; do
+            for suffix in Pool Updates Updates-test; do
```

Comment:
I think what I would do is say something like:
```bash
suffixes="Pool Updates"
if [[ want_test_updates = 1 ]]; then
    suffixes="$suffixes Updates-test"
fi

for suffix in $suffixes...
```

And remove the other logic at line 129